### chess square in ascii
uint8 file      # chess board file (column)
uint8 rank      # chess board rank (row)

### file ###
uint8 NoFile =  48  # '0' (ascii zero)
uint8 FileA  =  97  # 'a'
uint8 FileB  =  98  # 'b'
uint8 FileC  =  99  # 'c'
uint8 FileD  = 100  # 'd'
uint8 FileE  = 101  # 'e'
uint8 FileF  = 102  # 'f'
uint8 FileG  = 103  # 'g'
uint8 FileH  = 104  # 'h'

### rank ###
uint8 NoRank = 48  # '0' (ascii zero)
uint8 Rank1  = 49  # '1'
uint8 Rank2  = 50  # '2'
uint8 Rank3  = 51  # '3'
uint8 Rank4  = 52  # '4'
uint8 Rank5  = 53  # '5'
uint8 Rank6  = 54  # '6'
uint8 Rank7  = 55  # '7'
uint8 Rank8  = 56  # '8'
